rql/model/src/main/java/org/eclipse/ditto/rql/model/predicates/ast/SingleComparisonNode.java

```diff
@@ -78,7 +78,12 @@ public enum Type {
         /**
          * Represents a lower than or equals comparison.
          */
-        LIKE("like");
+        LIKE("like"),
+        
+         /**
+         * Represents a lower than or equals comparison case insensitive.
```

Comment:
Copy&paste error .. 

I have to admit that the `LIKE` also has a copy&paste error.
Could you please be so kind and adjust the Javadoc of both `LIKE` and `ILIKE`.

For LIKE e.g.: "Represents a string 'like' comparison, supporting wildcards '*' for multiple and '?' for a single character".
And for ILIKE mentioning the additional case insensitivity.